File: app/src/main/java/com/anokmik/tripassistant/user/login/LoginContract.java
package com.anokmik.tripassistant.user.login;

import com.anokmik.tripassistant.util.contract.presenter.PresenterValidator;
import com.anokmik.tripassistant.util.contract.view.ViewBack;

public interface LoginContract {

    interface View extends ViewBack {

    }

    interface Presenter extends PresenterValidator {

        void login();

    }

}

File: app/src/main/kotlin/com/anokmik/tripassistant/user/login/LoginContract.kt
package com.anokmik.tripassistant.user.login

import com.anokmik.tripassistant.util.contract.presenter.PresenterValidator
import com.anokmik.tripassistant.util.contract.view.ViewBack

interface LoginContract {

    interface View : ViewBack

    interface Presenter : PresenterValidator {

        fun login()

    }

}